### PR TITLE
OCPBUGS-26236: VolumeSnapshots data not displayed in PVC >  VolumeSnapshots tab

### PR DESCRIPTION
**Pull request by cyril-ui-developer**
### Before

Screenshot 2024-01-17 at 5 02 39 PM

### After

Screenshot 2024-01-17 at 10 17 00 AM

FYI: The Source column header is not displayed since version 4.10. I have opened a Jira ticket to address the issue because I need more understanding on how the `CAN_LIST_VSC` flag works.

### CODE REVIEW

**Comment by cyril-ui-developer** [comment]
Hi @bipuladh, This PR change is intended to fix this [Jira issue](https://issues.redhat.com/browse/OCPBUGS-26236). 
Since you worked on this code in the past, I wanted to double check with you on this change. I removed `snapshotStatusFilters` since the VolumeSnapshots tab should be displaying only matched PVC VolumeSnapshots,
 and  I am not sure why the `{ 'Snapshot Content': !canListVSC }` logic is use [here](https://github.com/openshift/console/blame/master/frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot.tsx#L206). PTAL, thanks.

PTAL, thanks in advance.

**Review by TheRealJon — CHANGES_REQUESTED** [review]
We shouldn't merge this without figuring out the missing column header.

**Comment by cyril-ui-developer** [comment]
> We shouldn't merge this without figuring out the missing column header.

I don't think so. Only if this issue is fixed before the other issue would be visible. 
So I prefer to address the missing col as a separate issue since it can be traced back to 4.10 version.

File: frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot.tsx [hunk 1]
```diff
@@ -232,18 +236,52 @@ const FilteredSnapshotTable: React.FC<FilteredSnapshotTable> = (props) => {
...
+    groupVersionKind: {
+      group: VolumeSnapshotModel.apiGroup,
+      kind: VolumeSnapshotModel.kind,
+      version: VolumeSnapshotModel.apiVersion,
+    },
+    isList: true,
+    namespaced: true,
+    namespace,
+  });
+  const rowFilters = [
```
[the remark this answers is not in the document]
Comment (cyril-ui-developer) [reply frontend/packages/console-app/src/components/volume-snapshot/volume-snapshot.tsx:252]: > Why are we redefining `volumeSnapshotStatus` ?

Not sure I understood what you meant here. I am not redefining `volumeSnapshotStatus`  but using it in the `rowFilter`. Pls explain further.

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**